### PR TITLE
naemon executable path

### PR DESCRIPTION
**Pull request by roengstrom**
Added ability to omit setting the executable path to the
naemon binary and the shared libraries. Also added possibility
to use absolute paths.

Signed-off-by: Robin Engström <robin.engstrom@op5.com>

### CODE REVIEW

**Comment by mattiasr** [comment]
Looks good to me

File: features/environment.py [hunk 1]
```diff
@@ -7,9 +7,22 @@
 
 
 def before_all(context):
-    os.environ['LD_LIBRARY_PATH'] = (
-        os.getcwd() + '/' + context.config.userdata['shared_libs_path']
-    )
+    # check if we have set relative paths in behave.ini
+    if 'shared_libs_path' in context.config.userdata:
+        shared_libs_path = ''
+        if context.config.userdata['shared_libs_path'][0] == '/':
```
Comment (tvestelind) [inline features/environment.py]: Wouldn't `os.path.abspath(context.config.userdata['shared_libs_path'])` achieve the same thing as all this?

File: features/environment.py [hunk 2]
```diff
@@ -7,9 +7,22 @@
...
+            shared_libs_path = context.config.userdata['shared_libs_path']
+        else:
+            shared_libs_path = os.getcwd() + '/' + \
+                context.config.userdata['shared_libs_path']
+        os.environ['LD_LIBRARY_PATH'] = shared_libs_path
+
+    if 'naemon_exec_path' in context.config.userdata:
+        if context.config.userdata['naemon_exec_path'][0] != '/':
```
Comment (tvestelind) [inline features/environment.py]: Same here for abspath

File: features/environment.py (hunk 1 above)
Comment (roengstrom) [inline features/environment.py]: That's way nicer, thanks! Updated code pushed.

**Pull request merged**